src/containers/share-modal/components/ShareDialog.tsx

```diff
@@ -72,23 +76,28 @@ export const ShareDialog = ({
             text={messages.twitter}
             onClick={onShareToTwitter}
             iconClassName={styles.twitterIcon}
-            textClassName={styles.twitterActionItemText}
+            textClassName={styles.twitterActionLabel}
           />
           {showTikTokShareAction ? (
             <ShareActionListItem
               leftIcon={<IconTikTok {...iconProps} />}
               text={messages.tikTok}
-              textClassName={cn(styles.tikTokActionItemText, {
-                [styles.tikTokActionItemTextDark]: isDarkMode()
-              })}
+              iconClassName={
+                isDarkMode() ? styles.tikTokIconDark : styles.tikTokIcon
+              }
+              textClassName={
+                isDarkMode()
```

Comment:
Nit: I think you can add this as a variable in colors.css, set values for both regular and dark mode, and then just use a single class with color: --var(tik-tok-icon-color) and not manually check for dark mode.

Reply:
ahh that makes a lot of sense. for context for this icon specifically i had to change the colors manually since the stems button is overriding the pre-defined fills in the svgs, in general, we dont need to reference the tikTok colors in css